package.json

```diff
@@ -23,10 +23,10 @@
     "cz": "env NODE_ENV=${NODE_ENV:-cli} git cz",
     "lint": "eslint src/*.js bin/*.js tests/*.mocha.js",
     "metapak": "metapak",
-    "mocha": "mocha tests/*.mocha.js",
+    "mocha": "mocha tests/*.mocha.js --timeout 10000",
```

Comment:
I guess that the need to add a timeout is an indicator of the fact that we should not support fill-rule attributes, at least, not by converting from svg to png and back to svg then.

Reply:
The timeout extend is for the async version of the sax library, it takes more time to process whe you have strokes for the conversion.